### ISSUE TITLE
Values of Location is not being updated in Editor

### ISSUE THREAD

**Issue by mohGhazala96** [dmbfm/unity-ar-gps-location-issues#57]
`       var newLocation = new Location()
        {
            Latitude = 31,
            Longitude = 29,
            Altitude = 0,
            AltitudeMode = AltitudeMode.Ignore
        };
        myObject.GetComponent<Hotspot>().Location.Longitude = longitude;`
when i print the value of myObject.GetComponent<Hotspot>().Location.Longitude it's prints the desired value but it doesn't change in the inspector!

**Comment by dmbfm** [comment]
Yes, it won't change in the inspector, that is expected. This getter/setter is used in code:
![image](https://user-images.githubusercontent.com/8743306/82507579-0f177e00-9ad9-11ea-9a9b-a276652a50e6.png)
which means you cant do something like `myObject.GetComponent<Hotspot>().Location.Longitude = longitude`, but instead `myObject.GetComponent<Hotspot>().Location = newLocation`.
